draft-ietf-tls-tls13.md

```diff
@@ -1574,10 +1601,6 @@ EncryptedExtensions
 : responses to any extensions which are not required in order to
   determine the cryptographic parameters. [{{encrypted-extensions}}]
```

Comment:
I think the two bits above should be switched; EncryptedExtensions should be before ServerConfiguration. The EncryptedExtensions message explicitly MUST come directly after the ServerHello, thus it is always before ServerConfiguration in order. I think the messages should be defined in logical use order, where possible, so that nothing implicitly contradicts anything if someone isn't reading carefully enough.